lib/checks/lists/dlitem.js

```diff
@@ -1,2 +1,21 @@
-var parent = axe.commons.dom.getComposedParent(node);
-return parent.nodeName.toUpperCase() === 'DL';
+const parent = axe.commons.dom.getComposedParent(node);
+
+if (!parent) {
+	return false;
+}
+
+const ALLOWED_ROLES = ['list'];
+const parentTagName = parent.nodeName.toLowerCase();
+const parentRole = (parent.getAttribute('role') || '').toLowerCase();
+
+if (parentTagName === 'dl') {
+	if (parentRole) {
+		if (ALLOWED_ROLES.includes(parentRole)) {
+			return true;
+		}
+		return false;
```

Comment:
Should check to see whether the other role is a valid ARIA role or otherwise it will not actually override the semantic

Reply:
Note: role='fake-role' fallback to semantic.